### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by pattems**
As promised in https://github.com/FreshRSS/FreshRSS/pull/2158#issuecomment-444164782, I have done a substantial rewrite & reorganization to the English administration documentation. This pulls stuff from the Readme, the documentation I'm proposing replacing, comments on pull requests, and my own personal experience with using and administering my instance of FreshRSS.

I haven't actually had the chance to test the commands on Debian 9 or Ubuntu, so that needs to be verified. I can do that, I just haven't had the chance to yet. The only todo is Apache config in 06_LinuxInstall.md... and honestly it might be the better course of action to just rip that section out and link to a good external tutorial for installing a LAMP stack.

The only other thing I'm unsure of is why footnotes, which I used in a couple of places, don't seem to be behaving correctly; they work fine in https://markdown-it.github.io/, but don't seem to actually convert to proper footnotes in the repository.

### CODE REVIEW

File: docs/en/admins/06_LinuxInstall.md [hunk 1]
```diff
@@ -69,6 +84,29 @@ Finally, symlink the public folder to the root of your web directory
...
+From the MySQL prompt (`MariaDB [(none)]>`), run the following commands, substituting \<username>, \<password>, and \<database_name> for real values.
+```
+CREATE USER '<username>'@'localhost' IDENTIFIED BY '<password>';
+CREATE DATABASE `databaseName`;
+GRANT ALL privileges ON `databaseName`.* TO 'userName'@localhost;
+FLUSH PRIVILEGES;
+QUIT;
+```
+
+A brief explination of the previous command block:
```
Comment (Frenzie) [inline docs/en/admins/06_LinuxInstall.md]: typo

File: docs/en/admins/06_LinuxInstall.md [hunk 2]
```diff
@@ -69,6 +84,29 @@ Finally, symlink the public folder to the root of your web directory
...
+* You first create a database user for FreshRSS to use.
+* Then you create a database for FreshRSS to store data in.
+* You grant permissions for the user you created to read, write, and modify the database.
+* Flushing privileges reloads the permissions, making the previous command to take effect.
```
Comment (Frenzie) [inline docs/en/admins/06_LinuxInstall.md]: typo

File: docs/en/admins/09_AccessControl.md [hunk 3]
```diff
@@ -21,3 +20,5 @@ Not using authentication on your server is dangerous, as anyone with access to y
 ## Hints
 
 You can switch your authentication method at any time by editing the `./data/config.php` file, on the line that begins `'auth_type'`.
+
+[^1]: See [Apache documentation](https://httpd.apache.org/docs/trunk/howto/auth.html)
```
Comment (Frenzie) [inline docs/en/admins/09_AccessControl.md]: See the Apache documentation

**Comment by Frenzie** [comment]
I just added a couple of random notes from a quick skim of some parts. I'll probably have some time to look at the actual content this weekend.

**Comment by pattems** [comment]
Thanks for the skim! Hopefully I'll have a chance to verify the Debian commands before the weekend. Also, is it really a commit from me without at least one typo? 😆

**Comment by Alkarex** [comment]
@pattems Could we resume this PR? (e.g. resolve conflicts) :-)

**Comment by pattems** [comment]
Sure, I'll get on it hopefully this evening.

**Comment by marienfressinaud** [comment]
I would like to do a bit of cleaning in the documentation but before that, I'd like to merge this PR to avoid complicated conflicts. Have you some time to work on it @pattems or do you prefer to let me finish it?

**Comment by Frenzie** [comment]
@marienfressinaud Btw, you mentioned the French docs being difficult to maintain, but I'm inclined to think the main problem lies less in the French translation itself and more in the lack of automation related to keeping things in sync. I recently discovered [po4a-gettextize](https://po4a.org/man/man1/po4a-gettextize.1.php) after looking for a way to solve that exact same problem.

That being said, maintaining the developer docs in French does seem like a wasted effort. Still, I'd be happy to do it in slightly broken French for practice purposes; the only problem is that way I'd place an editing burden on you guys. ;-)

```
# To generate an initial pot, do something like this
# po4a-gettextize -f latex -M utf-8 -m cv_fransdejonge_en-2019-05-28.tex -p bla.pot
#
# And then to generate a po file based on an existing translation
# po4a-gettextize -f latex -M utf-8 -m cv_fransdejonge.tex -l cv_fransdejonge.nl.tex -p cv_fransdejonge.nl.po


all: update build

build:
	po4a-translate -f latex -M utf-8 -m cv_fransdejonge.tex -p cv_fransdejonge.nl.po -l cv_fransdejonge.nl.tex
	lualatex cv_fransdejonge.tex
	lualatex cv_fransdejonge.tex
	lualatex cv_fransdejonge.nl.tex
	lualatex cv_fransdejonge.nl.tex

update:
	po4a-updatepo -f latex -M utf-8 -m cv_fransdejonge.tex -p cv_fransdejonge.pot
	po4a-updatepo -f latex -M utf-8 -m cv_fransdejonge.tex -p cv_fransdejonge.nl.po
```

**Comment by marienfressinaud** [comment]
I mentioned that mainly because it is time consuming and we don't even take this time for English documentation (well, we take it more and more). It's totally discouraging me just to think that I need to write documentation twice :sweat_smile: 

(I think your comment belongs to #2491 :wink:)

File: docs/en/admins/06_LinuxInstall.md (hunk 2 above)
Comment (marienfressinaud) [reply docs/en/admins/06_LinuxInstall.md]: I don't see the typo here :sweat_smile:

Comment (Frenzie) [reply docs/en/admins/06_LinuxInstall.md]: *To* take effect is ungrammatical, at least in a subordinate clause like this.
```suggestion
* Flushing privileges reloads the permissions, which makes the previous command take effect.
```

Comment (marienfressinaud) [reply docs/en/admins/06_LinuxInstall.md]: Ah, I knew I should have listened in class!

**Comment by marienfressinaud** [comment]
I made some improvements to the documentation but I don't want to spend too much time on this. The PR already modify a lot of files, I'd like to merge it quickly (merging `dev` was a bit complicated). I'll spend some time to reorganize and improve the documentation (see #2491) once I'll be back from my vacations (in one month).

I suggest to merge the PR and if you note mistakes, just comment in #2491, I'll take care of them at the end of October :)

**Review by marienfressinaud — APPROVED** [review]

File: docs/en/admins/04_Updating.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Backing Up and Updating FreshRSS
+
+The following is general procedure; [specific commands for Linux may be found here](07_LinuxUpdate.md).
+
+## Backing Up
```
Comment (Alkarex) [inline docs/en/admins/04_Updating.md:5]: We should add the option to import/export by SQLite. There is another documentation issue left open for that

File: docs/en/admins/06_LinuxInstall.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
+# Installation on Debian 9/Ubuntu 16.04
+
+This tutorial will give you step-by-step commands to install the latest stable release of FreshRSS with Apache and MySQL using git. It's always recommended that you [backup your installation](05_Backup.md) before updating
```
Comment (Alkarex) [inline docs/en/admins/06_LinuxInstall.md:3]: We should add a link to the Docker option

**Pull request merged**